core/trino-spi/pom.xml

```diff
@@ -212,6 +212,7 @@
                                 </item>
                                 <!-- Backwards incompatible changes since the previous release -->
                                 <!-- Any exclusions below can be deleted after each release -->
+                                <!-- Remember to notify to all Trino maintainers before merging backward incompatible SPI changes -->
```

Comment:
This is not a policy we currently have.  Please bring this up with the maintainers group if you would like to add this policy.